src/modules/traceContext.js

```diff
@@ -35,6 +36,38 @@ class TraceContext {
                 }
             }).then((response) => response.arrayBuffer()));
     };
+
+    /**
+     * Add a tag to this trace.
+     *
+     * @param {string} tag Text to be added as tag, usually a single word
+     * @returns The promise of the response
+     */
+    addTag = (tag) => {
+        return this.sessionManager.gatekeeper(`/projects/${this.collectionId}/traces/${this.traceUid.traceUid}/tags/${window.encodeURIComponent(tag)}`, {
+            method: 'PUT',
+            headers: {
+                'Hansken-Project-Refresh': 'refresh',
```

Comment:
What is the default for this on the server side, and should we make this overridable here, as an optional argument to this method?

Reply:
The server default is `no-refresh`, but in frontends it is `refresh` by default. In python (batches) the default is no-refresh I think...

Reply:
I updated the code so the refresh is configurable btw